src/coreclr/vm/jitinterface.cpp

```diff
@@ -9323,6 +9323,431 @@ CorInfoTypeWithMod CEEInfo::getArgType (
     return result;
 }
 
+// NOTE:
+// Although this is only used for LoongArch64-ABI now,
+// maybe it can be used for other architecture for getting ABI-info
```

Comment:
I think it is very unlikely that this would be reusable for other architectures (except for RiscV that it is copied from).

Reply:
It may look better to model this same way as `getSystemVAmd64PassStructInRegisterDescriptor`: make it very clear even in the method name that it is LoongArch specific method.

Reply:
rename is as `getSystemVLoongarch6464PassStructInRegisterDescriptor` ？

Reply:
I would make it `getLoongArch64PassStructInRegisterFlags`. 

I do not think you use `SystemV` in the Longarch ABI name like what it is on amd64 (https://refspecs.linuxbase.org/elf/x86_64-abi-0.99.pdf).

Reply:
> I would make it `getLoongArch64PassStructInRegisterFlags`.
> 
> I do not think you use `SystemV` in the Longarch ABI name like what it is on amd64 (https://refspecs.linuxbase.org/elf/x86_64-abi-0.99.pdf).

Thanks